src/bandersnatch/verify.py

```diff
@@ -241,7 +254,7 @@ async def metadata_verify(config: ConfigParser, args: Namespace) -> int:
 
     logger.info(f"Starting verify for {mirror_base_path} with {workers} workers")
     try:
-        json_files = await loop.run_in_executor(executor, os.listdir, json_base)
+        json_files = list(x.name for x in json_base.iterdir())
```

Comment:
Can't this block the async event loop?

Reply:
I'm not so familliar with this, will try to do that!

Reply:
I didnt find a way to achive that , listdir() yield a `generator` like `range` , If I have to do this in another executor , I have to wrap this up and call it with executor, which seems to be unneccessary ? What do you think?